CHANGELOG.md

```diff
@@ -1,42 +1,47 @@
-# 1.3.1 (2022-09-13)
```

Comment:
Any reason why you're removing these dates?

Reply:
Improved readability, simplified release process.  The release dates are available via the repo tags for those who are interested.